Buttons/Toggle buttons_01/code/style.css

```diff
@@ -0,0 +1,96 @@
+@import url(https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0);
+
+*{
+    box-shadow: none;
+}
+
+.parent{
+    font-family: 'Poppins', sans-serif;
+    margin: 0;
+    width: 100%;
+    height: 100vh;
+    background-color: #d1dad3;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 17px;
+    font-weight: 400;
+}
+
+.container{
+    max-width: 1000px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    flex-wrap: wrap;
+}
+
+.switch-holder{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 10px 20px;
+    border-radius: 10px;
+    margin-bottom: 30px;
+    box-shadow: -8px -8px 15px rgba(255,255,255,0.7), 
+                10px 10px 10px rgba(0,0,0,0.3),
+                inset 8px 8px 15px rgba(255,255,255,0.7),
+                inset 10px 10px 10px rgba(0,0,0,0.3);
+}
+
+.switch-label{
+    width: 150px;
+}
+
+.switch-label i{
+    margin-right: 5px;
+}
+
+.switch-toggle{
+    height:40px;
+}
+
+.switch-toggle input[type="checkbox"]{
+    position: absolute;
+    opacity: 0;
+    z-index: -2;
+}
+
+.switch-toggle input[type="checkbox"]+label{
+    position: relative;
+    display: inline-block;
+    width: 100px;
+    height: 40px;
+    border-radius: 20px;
+    margin: 0;
+    cursor: pointer;
+    box-shadow: inset -8px -8px 15px rgba(255,255,255,0.6),
+                inset 10px 10px 10px rgba(0,0,0,0.25);
+}
+
+.switch-toggle input[type="checkbox"]+label::before{
+    position: absolute;
+    content: "OFF";
+    font-size: 13px;
+    text-align: center;
+    line-height: 25px;
+    top: 8px;
+    left: 8px;
+    width: 45px;
+    height: 25px;
+    border-radius: 20px;
+    background-color: #d1dad3;
+    box-shadow: -3px -3px 5px rgba(255,255,255,0.5),
+                3px 3px 5px rgba(0,0,0,0.25);
+    transition: 0.3s ease-in-out;
+}
+
+.switch-toggle input[type="checkbox"]:checked+label::before{
+    left: 50%;
+    content: "ON";
+    color: #fff;
```

Comment:
Change the color from white to black for more visibility . I think #1f1f1f will be better